### PR TITLE
allow all resolutions if there is no volume annotation in the nml

### LINKED ISSUE #6063: Uploading annotation zip with empty volume layers leads to error
## Context
When uploading an annotation zip-file which includes empty volume tracings (which is produced e.g. when downloading an annotation without brushing, or via wk-libs), then I get the following error:
```
Unsuccessful WS request to https://webknossos.org/tracings/volume/5aa9f1e2-fa19-467c-8d1b-85d776548aa1/initialData (ID: 551).Status: 400. Response: {"messages":[{"error":"Resolution restrictions result in zero resolutions"}]}
```

This should not happen.

## Steps to Reproduce the bug
1. Drag'n'Drop the following annotation zip into webknossos
  [l4dense_motta_et_al_nuclei__explorational__jstriebel__b3eddd.zip](https://github.com/scalableminds/webknossos/files/8108776/l4dense_motta_et_al_nuclei__explorational__jstriebel__b3eddd.zip)
2. The error mentioned above is shown, no annotation opens :crying_cat_face:

### PR DESCRIPTION
**Pull request by leowe**
- When uploading a NML check whether there are any volume annotations, if there are none, allow all resolutions
- add `leowe` to slack notifications
- fix small language mistake
- change `logger.error` to `logger.warn` for filesystem checking for the data set indexing

### URL of deployed dev instance (used for testing):
- https://emptytracing.webknossos.xyz/

### Steps to test:
- Open any dataset (e.g. `l4_sample`)
- Create a empty annotation, download it, and then drag and drop it in the view (or use [this](https://github.com/scalableminds/webknossos/files/8262647/l4_sample__explorational__suser__bb74f7.zip) empty annotation)
- A new annotation should open without an error message

### Issues:
- fixes #6063 

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [x] Needs datastore update after deployment
- [x] Ready for review

### CODE REVIEW

File: webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala [hunk 1]
```diff
@@ -186,11 +186,15 @@ class VolumeTracingService @Inject()(
   def initializeWithData(tracingId: String,
                          tracing: VolumeTracing,
                          initialData: File,
-                         resolutionRestrictions: ResolutionRestrictions): Box[Set[Vec3Int]] = {
+                         resolutionRestrictions: ResolutionRestrictions): Fox[Set[Vec3Int]] = {
     if (tracing.version != 0L) {
       return Failure("Tracing has already been edited.")
     }
 
+    val resolutionSet = resolutionSetFromZipfile(initialData)
```
Comment (fm3) [inline webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala:194]: This unpacks the zipfile an additional time for each upload. Could the empty check also happen on the result from below?

**Review by fm3 — COMMENTED** [review]
Cool stuff, I like that you also fixed a few small things along the way! :D  
I added a suggestion for performance optimization.

File: webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala (hunk 1 above)
Comment (leowe) [reply webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala]: Good point, I changed it.

File: webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala [hunk 2]
```diff
@@ -202,6 +202,9 @@ class VolumeTracingService @Inject()(
         saveBucket(dataLayer, bucketPosition, bytes, tracing.version)
       }
     }
+    // if none of the tracings contained any volume data. do not save buckets, use full resolution list
```
Comment (fm3) [inline webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala:205]:
```suggestion
    // if none of the tracings contained any volume data, use the dataset’s full resolution list
```

**Review by fm3 — APPROVED** [review]
LGTM 🚀

**Pull request merged**